docs/UserGuide.md

```diff
@@ -343,8 +345,7 @@ _Details coming soon ..._
 ## FAQ
 
 **Q**: How do I transfer my data to another Computer?<br>
-**A**: Install the app in the other computer and overwrite the empty data file it creates with the file that contains
-the data of your previous AddressBook home folder.
+**A**: Install the app in the other computer and overwrite the empty data file it creates with the file that contains the data of your previous SpleetWaise home folder.
```

Comment:
```suggestion
**A**: Install the app in the other computer and overwrite the sample data file it creates with the file that contains the data of your previous SpleetWaise home folder.
```

Reply:
I believe when we just install the app, there isnt even the data file. we should test out, if we copy over the data files to the new app location, it should work accordingly.